.github/workflows/ci.yml

```
@@ -2,42 +2,46 @@
-        run: npm run rebuild:sqlite
-      - name: Sanity run
-        run: node -e "require('better-sqlite3'); console.log('better-sqlite3 OK')"
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)